utils_linux.go

```diff
@@ -56,6 +56,7 @@ func loadFactory(context *cli.Context) (libcontainer.Factory, error) {
 		}
 	}
 
+	intelrdt.InitIntelRdt()
 	intelRdtManager := libcontainer.IntelRdtFs
 	if !intelrdt.IsCATEnabled() && !intelrdt.IsMBAEnabled() {
 		intelRdtManager = nil
```

Comment:
Looks like this can be simplified by moving init and feature check to the function that `libcontainer.IntelRdtFs` returns. Can you look into it @xiaochenshen ?

Reply:
> 
> 
> Looks like this can be simplified by moving init and feature check to the function that `libcontainer.IntelRdtFs` returns. Can you look into it @xiaochenshen ?

@kolyshkin Thank you. I will move init and feature check into `libcontainer.IntelRdtFs` and simplify `IntelRdtManager` initialization in `LinuxFactory`.